cmd/kola/options.go

```diff
@@ -171,13 +171,29 @@ func syncOptions() error {
 		return err
 	}
 
-	image, ok := kolaDefaultImages[kola.QEMUOptions.Board]
-	if kola.QEMUOptions.Distribution == "cl" && !ok {
-		return fmt.Errorf("unsupport board %q", kola.QEMUOptions.Board)
+	var cosaBuild *cosa.Build
+	if kola.Options.CosaBuild != "" {
+		f, err := os.Open(kola.Options.CosaBuild)
+		if err != nil {
+			return err
+		}
+		defer f.Close()
+		dec := json.NewDecoder(f)
+		// FIXME enable this to prevent schema regressions https://github.com/coreos/coreos-assembler/pull/1059
+		// dec.DisallowUnknownFields()
+		var tmpBuild cosa.Build
+		if err := dec.Decode(&tmpBuild); err != nil {
+			return err
+		}
+		cosaBuild = &tmpBuild
 	}
 
 	if kola.QEMUOptions.DiskImage == "" {
-		kola.QEMUOptions.DiskImage = image
+		if cosaBuild != nil {
+			kola.QEMUOptions.DiskImage = filepath.Join(filepath.Dir(kola.Options.CosaBuild), cosaBuild.Images.QEMU.Path)
+		} else {
+			panic("No qemu disk image provided")
```

Comment:
Rather than a `panic` you can just return an error and it will still directly exit (https://github.com/coreos/mantle/blob/master/cmd/kola/kola.go#L95).

Reply:
Thanks, fixed!